python/datasources/cawp_time.py

```diff
@@ -1,20 +1,26 @@
-from typing import List
+import pandas as pd
+from ingestion.merge_utils import ACS_EARLIEST_YEAR, ACS_LATEST_YEAR
+from ingestion.standardized_columns import Race
+from ingestion.dataset_utils import (generate_pct_rel_inequity_col,
+                                     zero_out_pct_rel_inequity)
+from ingestion import gcs_to_bq_util, merge_utils
+import ingestion.standardized_columns as std_col
 from datasources.data_source import DataSource
 from ingestion.constants import (
     NATIONAL_LEVEL, STATE_LEVEL,
     STATE_LEVEL_FIPS_LIST,
     TERRITORY_FIPS_LIST,
     US_ABBR, US_FIPS, US_NAME,
     TERRITORY_POSTALS,
-    RACE
 )
-import ingestion.standardized_columns as std_col
-from ingestion import gcs_to_bq_util, merge_utils
-from ingestion.dataset_utils import (generate_pct_rel_inequity_col,
-                                     zero_out_pct_rel_inequity)
-from ingestion.standardized_columns import Race
-from ingestion.merge_utils import ACS_EARLIEST_YEAR, ACS_LATEST_YEAR
-import pandas as pd
+from typing import Literal, cast, List
+
+SEX_RACE_AGE_TYPE = Literal["sex", "age", "race"]
```

Comment:
Fancy

Reply:
I think we should slowly be typing all our functions; do you support that? I feel like now that I'm used to typescript it seems so silly to be able to break a function by sending in something of the wrong type. Python's typing isn't as easy to work with but seems ok for stuff like this where it needs to be specific keywords and it's nice when it populated in the IDE hints

Reply:
Typescript is so great to use because of typing, but I always struggle to type with Python correctly. So I'll probably implement this in my HIV PR.